src/wp-admin/includes/ajax-actions.php

```diff
@@ -3975,18 +3975,34 @@ function wp_ajax_crop_image() {
 
 			$size       = wp_getimagesize( $cropped );
 			$image_type = ( $size ) ? $size['mime'] : 'image/jpeg';
+			/** @var WP_Post $original_attachment */
+			$original_attachment = get_post( $attachment_id );
+			$has_title           = 0 < mb_strlen( trim( $original_attachment->post_title ) );
```

Comment:
```suggestion
			$has_title           = mb_strlen( trim( $original_attachment->post_title ) ) > 0;
```
A Yoda condition here degrades readability, per the WPCS(https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/)

Reply:
Thank you for the feedback! Fixed in fdaa748a589a8cb0ab7e21b624e38f35284027df